src/chroot_script

```diff
@@ -212,6 +212,25 @@ echo "mapping wlan1" >> /etc/network/interfaces
 echo "  script /root/bin/map_iface" >> /etc/network/interfaces
 echo "source /boot/octopi-network.txt" >> /etc/network/interfaces
 
+# allow wifi network configuration via /boot/octopi-wifi.txt
+mv /etc/wpa_supplicant/wpa_supplicant.conf /boot/octopi-wifi.txt
+ln -s /boot/octopi-wifi.txt /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
Thumbs-up on keeping the file name `wpa_supplicant.conf`. However I'm not sure if adding a folder `advanced` is necessary. `wpa_supplicant.conf` isn't harder to understand and change than most other files currently in `/boot`. Adding an `advanced` folder just for this file seems overkill.